generator/autogenlist.ts

```diff
@@ -454,6 +454,20 @@ const autogenlist: AutogenlistConfig[] = [
         basePath: 'relay/resource-manager',
         namespace: 'Microsoft.Relay',
     },
+    {
+        basePath: 'recoveryservicessiterecovery/resource-manager',
+        namespace: 'Microsoft.RecoveryServices',
+        suffix: 'SiteRecovery',
+    },
+    {
+        basePath: 'recoveryservicesbackup/resource-manager',
+        namespace: 'Microsoft.RecoveryServices',
+        suffix: "Backup"
+    },
+    {
+        basePath: "recoveryservices/resource-manager",
+        namespace: "Microsoft.RecoveryServices"
+    },
```

Comment:
Will the PR-driven swagger pipeline you added respect this file?

Reply:
yes, it will